tests/unit/Core/System/Snippet/Files/SnippetFileLoaderTest.php

```diff
@@ -212,27 +212,51 @@ public function testLoadBaseSnippetsFromPlugin(): void
 
         $snippetFileLoader->loadSnippetFilesIntoCollection($collection);
 
-        static::assertCount(2, $collection);
+        static::assertCount(4, $collection);
+        static::assertCount(2, $collection->getSnippetFilesByIso('de-DE'));
 
-        $snippetFile = $collection->getSnippetFilesByIso('de-DE')[0];
-        static::assertEquals('storefront.de-DE', $snippetFile->getName());
+        $adminSnippetFile = $collection->getSnippetFilesByIso('de-DE')[0];
+        static::assertEquals('de-DE', $adminSnippetFile->getName());
```

Comment:
```suggestion
        static::assertSame('de-DE', $adminSnippetFile->getName());
```

please use `assertSame` where possible instead, as it is type safe. Please fix also the following occurrences